src/util/importUtil.js

```diff
@@ -8,12 +8,12 @@ function getNameFromCjsRequire(init) {
 }
 
 
-const isFullLodashImport = str => /^lodash(-es)?$/.test(str)
+const isFullLodashImport = str => /^lodash(-es)?(\/fp)?$/.test(str)
```

Comment:
This solution would be problematic.
`importUtil` is used for `LodashContext`, which would mean that functions from `lodash/fp` would be recognized as Lodash functions by the plugin, and the rules don't consider `lodash/fp`, so there would be a lot of unforeseen consequences because of parameter order, currying etc.
That's why [`eslint-plugin-lodash-fp`](https://github.com/jfmengels/eslint-plugin-lodash-fp) is a separate plugin.
I would solve this by a RegExp that matches Lodash functions **but not `lodash/fp` functions**.